site/src/api/typesGenerated.ts

```diff
@@ -291,6 +291,13 @@ export interface UpdateRoles {
   readonly roles: string[]
 }
 
+// From codersdk/templates.go:34:6
+export interface UpdateTemplateMeta {
+  readonly description: string
+  readonly max_ttl_ms: number
+  readonly min_autostart_interval_ms: number
```

Comment:
These should be transformed to optional (`?`) fields via the `omitempty` suggestion above.